### PR TITLE
updated-installation-guidelines

### PR DESCRIPTION
**Pull request by carrisunrio**
Targets #{1028}

### Description
Changes made to the installation guidelines as required to increase the readability and effective usage of the guidelines

### Checklist:

- [ ✔] I have performed a self-review of my own code.

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
+
+0. Navigate to the right diretory where your project will be locally saved
+    - For WAMP: `C:\wamp64\www\`
+    - For XAMPP: `C:/xampp/htdocs/`
+    - For MAMP(Mac OS): `/Application/MAMP/htdocs/`
+
 1. Clone this repository and move to `portal` directory
    ```sh
-   git clone https://github.com/coloredcow-portal/portal
+   git clone git clone https://github.com/coloredcow/portal
```
Comment (rathorevaibhav) [inline docs/installation.md:12]:
```suggestion
   git clone https://github.com/coloredcow/portal
```

File: docs/installation.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
 ## Installation Guidelines :rocket:
 
+Before you start following the guidelines, make sure to go through the [prerequisites](./docs/prerequisites.md) to get started.
```
Comment (rathorevaibhav) [inline docs/installation.md:3]:
```suggestion
Before you start following the guidelines, make sure to go through the [prerequisites guide](./docs/prerequisites.md) to install the required tools and packages on your machine.
```

File: docs/installation.md [hunk 3]
```diff
@@ -1,8 +1,15 @@
...
+
+0. Navigate to the right diretory where your project will be locally saved
+    - For WAMP: `C:\wamp64\www\`
+    - For XAMPP: `C:/xampp/htdocs/`
+    - For MAMP(Mac OS): `/Application/MAMP/htdocs/`
```
Comment (rathorevaibhav) [inline docs/installation.md:8]:
```suggestion
1. Navigate to the right diretory where your project will be locally saved
    - For WAMP: 
        ```sh
        cd C:\wamp64\www\
        ```
    - For XAMPP: 
        ```sh
        cd C:/xampp/htdocs/
        ```
    - For MAMP(Mac OS): 
        ```sh
        cd /Application/MAMP/htdocs/
        ```
```

Comment (rathorevaibhav) [reply docs/installation.md:8]: Also, what's the difference between backslash `\` and forward slash `/` here in the directory names? Do they work differently?

File: docs/installation.md [hunk 4]
```diff
@@ -27,10 +34,12 @@
    ```
 
 
-4. Copy `.env.example` as `.env`
-
+4. Copy and paste `.env.example` file as `.env` file there itself
```
Comment (rathorevaibhav) [inline docs/installation.md:37]:
```suggestion
4. Make a copy of the `.env.example` file in the same directory and save it as `.env`:
```

File: docs/installation.md [hunk 5]
```diff
@@ -27,10 +34,12 @@
...
+     ```sh
+    cp .env.example .env
+    ```
 
-5. Run the following command to add a key
+5. Run the following command to add a key:
```
Comment (rathorevaibhav) [inline docs/installation.md:42]:
```suggestion
5. Run the following command to add the Laravel application key:
```

File: docs/installation.md [hunk 6]
```diff
@@ -45,14 +54,14 @@
     ```
 
     2. Database configurations
-     Make sure you have a database created in your local server. For more info check this [link](https://www.youtube.com/watch?v=4geOENi3--M).
+     Make sure you have a database created in your local server. For more info check this [link](https://www.youtube.com/watch?v=k9yJR_ZJbvI&ab_channel=1BestCsharpblog)(skip to 0:21) for creating an empty database and this [link](https://www.youtube.com/watch?v=4geOENi3--M)(relevant timestamp periods : 2:00-2:42 & 4:20-5:40) to continue with the database configuration.
```
Comment (rathorevaibhav) [inline docs/installation.md:57]:
```
2. Database configurations
    - Create a database in your local server. Check out [this link](URL) and skip to 0:21.
    - Configure your Laravel app with the right DB settings. Check out [this link](URL). Relevant parts are 2:00-2:42 and 4:20-5:40.
```

File: docs/installation.md [hunk 7]
```diff
@@ -45,14 +54,14 @@
...
     ```sh
-    DB_CONNECTION=
-    DB_HOST=
-    DB_PORT=
-    DB_DATABASE=
-    DB_USERNAME=
+    DB_CONNECTION=mysql
+    DB_HOST=127.0.0.1
+    DB_PORT=3306
+    DB_DATABASE=homestead
```
Comment (rathorevaibhav) [inline docs/installation.md:63]:
```suggestion
    DB_DATABASE=portal
```

File: docs/installation.md [hunk 8]
```diff
@@ -124,7 +153,7 @@
             ```
         - Restart XAMPP. Next, open this url in your browser: http://portal.test
     
-    2. For MAMP(Mac OS)
+    3. For MAMP(Mac OS):
```
Comment (rathorevaibhav) [inline docs/installation.md:156]:
```suggestion
    3. For MAMP(macOS):
```

**Review by rathorevaibhav — CHANGES_REQUESTED** [review]
@carrisunrio some minor edits and questions.

Also, the file `package-lock.json` is not relevant for this task. Can you remove it from the PR?

File: docs/installation.md (hunk 3 above)
Comment (carrisunrio) [reply docs/installation.md:8]: For MAMP, it is different, rest of the two OSes use a similar kind of slash i.e; backslash \

**Comment by rathorevaibhav** [comment]
@carrisunrio there are some merge conflicts in here. Can you resolve them so that the team can do the final review?

**Review by rathorevaibhav — APPROVED** [review]

**Pull request merged**